iOS/Flows/Conversation/ConversationViewController+Extension.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ConversationViewController+Updates.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 10/15/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+extension ConversationViewController {
+
+    func setupCompletionHandlers() {
```

Comment:
Moved these to an extension because it was getting crowded. Also grouped the functions by type.